docs/docs/zh/guide/project/workflow-definition.md

```diff
@@ -75,23 +75,48 @@
   * 收件人：选择通知策略||超时报警||发生容错时，会发送流程信息或告警邮件到收件人列表。
   * 抄送人：选择通知策略||超时报警||发生容错时，会抄送流程信息或告警邮件到抄送人列表。
   * 启动参数: 在启动新的流程实例时，设置或覆盖全局参数的值。
-  * 补数：包括串行补数、并行补数 2 种模式，日期可以通过选择或者手动进行输入。串行补数：指定时间范围内，从开始日期至结束日期依次执行补数，依次生成N条流程实例；并行补数：指定时间范围内，多天同时进行补数，同时生成 N 条流程实例。手动输入日期：手动输入以逗号分割日期格式为 `yyyy-MM-dd HH:mm:ss` 的日期。日期选择：通过前端选择日期。
+  * 补数：指运行指定日期范围内的工作流定义，根据补数策略生成对应的工作流实例，补数策略包括串行补数、并行补数 2 种模式，日期可以通过页面选择或者手动输入。
+   
+    * 串行补数：指定时间范围内，从开始日期至结束日期依次执行补数，依次生成多条流程实例；点击运行工作流，选择串行补数模式：例如从7月 9号到7月10号依次执行，依次在流程实例页面生成两条流程实例。
+  
+    ![workflow-serial](../../../../img/new_ui/dev/project/workflow-serial.png)
+    
+    * 并行补数： 指定时间范围内，多天同时进行补数，同时生成多条流程实例。手动输入日期：手动输入以逗号分割日期格式为 `yyyy-MM-dd HH:mm:ss` 的日期。点击运行工作流，选择并行补数模式：例如同时执行7月9号到7月10号的任务，同时在流程实例页面生成两条流程实例。
```

Comment:
It can be noted here that when the workflow definition execution strategy is serial wait or serial priority, the workflow instance is executed according to the execution strategy.
For example: parallel complement execution from July 9th to July 10th. The execution strategy is defined for the workflow of serial waiting. The execution of the workflow instance is that only one workflow instance is executed at the same time, and the state of the next instance is serial waiting. Attached example image. The en md is the same.